.circleci/config.yml

```diff
@@ -100,6 +100,18 @@ jobs:
           name: check optimized images
           command: make lint-assets
 
+  check-lockfiles:
+    docker:
+      - image: cimg/ruby:2.7-node
+    environment:
+      SKIP_INSTALL: true
+    steps:
+      - checkout
+      - install-dependencies
+      - run:
+          name: check lockfile differences
+          command: make lint-assets
```

Comment:
should this be `validate-lockfiles`?

```suggestion
          command: make validate-lockfiles
```

Reply:
> should this be `validate-lockfiles`?

Bah, yes, sloppy copy-paste on my part 😩 

Thanks.